src/main/java/frc/robot/subsystems/loader/LoaderSS.java

```diff
@@ -25,7 +25,7 @@ public LoaderSS(LoaderConstants constants) {
      */
     public void setDesiredVelocity(double desiredVelocity) {
         if (!overridden)
-            motor.set(ControlMode.Velocity, desiredVelocity);
+            motor.setVoltage(constants.FEEDFORWARD_CONSTANTS.mCoef * desiredVelocity + Math.signum(desiredVelocity) * constants.FEEDFORWARD_CONSTANTS.bCoef);
```

Comment:
Why are we adding a f to the loader? we agreed that it makes close to no difference coz the shooter moves so fast